Assets/_Functionalities/ObjectInformation/Scripts/HiddenBagIds.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Netherlands3D.Twin
+{
+    [CreateAssetMenu(menuName = "Netherlands3D/Data/BagIdsHiddenData", fileName = "BagIdsHidden", order = 0)]
+    public class HiddenBagIds : ScriptableObject
+    {         
+        [SerializeField] public List<string> bagIds = new List<string>();
```

Comment:
Kan dit niet beter gewoon een lijst van strings zijn waar het nodig is? of is dit in meerdere scripts nodig?